tools-local/Microsoft.ML.CodeAnalyzer/Microsoft.ML.CodeAnalyzer.csproj

```diff
@@ -1,12 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard1.3</TargetFramework>
```

Comment:
An other option (if you need dependencies) would be to target the concrete TFM (either net4x for desktop msbuild, or netcoreapp2.x for CLI-based msbuild, or cross-target to support both) and direct the SDK to copy those dependencies to the output folder.  I believe the [default ](https://github.com/dotnet/sdk/blob/581f5371dac022ad2b3e5bcdbb2e576bd73b5c4a/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets#L61-L78) is to not copy dependencies for netcoreapp and netstandard.

Reply:
Hi @ericstj , I don't *think* I need any dependencies, at least, aside from those directly needed by any C# Roslyn analyzer. The original analyzer I was porting was actually .NET Standard 1.3 anyway, so this is, I guess, fine?

(TBH I don't actually have any idea why this error was occurring. This problem of IDE1003 was only reproing on my home computer, not my work computer. Which makes no sense to me, since AFAIK my Visual Studio installations on the two are more or less the same.)

Reply:
I see, then yeah this is fine for now.  `netstandard1.3` had inbox support without any out-of-band facades on `net46`.  That's why it works well here.

`netstandard1.5` had some out-of-band facades required for support on [`net461`-`net47`] which is likely why your home computer was failing.  It probably didn't have `net471` or later, but your work machine does.